### DISCUSSION TITLE
When osu!lazer will migrate to .NET 6.0?

### DISCUSSION THREAD

**Discussion by BlitDev** [ppy/osu#16242] [Ideas] [answered]
When building from source, there are problems on some Linux distributions (including Arch), the compiler warns that osu!lazer cannot be built using .NET 6.0 SDK, but there is no .NET 5.0 SDK in repositories

**Comment by turbedi** [comment, marked as answer]
Duplicate: https://github.com/ppy/osu/discussions/15544
You can find instructions on how to manually install multiple .NET SDK versions on Arch Linux [here](https://wiki.archlinux.org/title/.NET#Install_multiple_versions_manually):

**Comment by peppy** [comment]
Please use the provided appimages if you are unable to build from source due. Building from source is not something we support apart for development purposes.
